### DISCUSSION TITLE
How to determine statistically significant marker peak across batches?

### DISCUSSION THREAD

**Discussion by Zhongjiacheng1** [GreenleafLab/ArchR#1602] [Questions / Documentation]
Greetings, dear developers
I wondered whether it is possible to incorporate batch effect information into a regression framework when identifying differentially accessible peaks.

e.g. In Seurat, one may use "orig.ident" as a latent variable in findmarker function to regress out the batch effect and determine differential gene expression.

It seems to me that, the "bias" parameter in "getMarkerFeatures" could do the trick, but I am not sure.

**Comment by AnjaliC4** [comment]
Hi @rcorces,

I have a similar question, does it make sense to use a categorical variable such as, as.numeric("batch") in the "bias" parameter of getMarkerFeatures?  I assume it would match each cell in the  "useGroups" with "bgdGroups" based on the batch it is coming from which could presumably avoid effects due to batch differences. Does it make sense? I tried to understand the code but it was a bit difficult for me to follow.

**Reply by rcorces** [reply]
In my opinion, the best way to do this is to do your differential comparisons on sample-centric pseudobulks and (as needed) incorporate batch into a more standard regression framework using conventional bulk ATAC-seq analysis. We are in the process of updating differential comparisons to make pseudobulk-based differentials easier. For now, you would use the function `getGroupSE()` or the new function `getPBGroupSE()` to export pseudobulked `SummarizedExperiment` objects. The rest of the differential testing would have to be done using your own code.


`getPBGroupSE()` is only available on `dev` at the moment.
https://github.com/GreenleafLab/ArchR/blob/c44b50c56323d5995fb3d2fb28a9072255fec644/R/GroupExport.R#L143-L230

**Reply by AnjaliC4** [reply]
Thanks @rcorces. We actually did perform differential analysis with a pseudobulk approach and incorporated batch in the model as you suggested. I have been more wondering about ‘bias’ parameter for Motif matrix. Since Im testing on z-scores of motif deviation (useseqnames=z) - can I add ‘batch’ in the bias parameter along with nFrags? I would like to get a sense of how ‘bias’ would work for categorical variables? Thanks very much!

**Reply by rcorces** [reply]
I have never done this and am not sure how it would perform

**Reply by AnjaliC4** [reply]
It seems like it should match group1 cell to group2 cell from the same batch. But I’m also unsure of this.
